tests/ceph_installer/test_ansible.py

```diff
@@ -146,6 +147,22 @@ def run(ceph_cluster, **kw):
 
     LOG.info("Ceph ansible version " + ceph_installer.get_installed_ceph_versions())
 
+    # Check if ansilbe.log file has any logs initially
+    if custom_log:
+        ansible_log_file, _ = ceph_installer.exec_command(
+            cmd=f"cat {ansible_dir}/ansible.cfg | grep log_path | cut -d \" \" -f3 | tr -d '\n\r'".format(
+                ansible_dir=ansible_dir
+            )
+        )
+        file_size_before, err = ceph_installer.exec_command(
+            cmd=f"du \"{ansible_log_file}\" | cut -f1 | tr -d '\n\r'".format(
```

Comment:
When using `f` strings, the right approach would be 

`cmd=f"du {ansible_log_file} | cut -f1 | tr -d '\n\r'"`

Reply:
Updated all the changes.